### DISCUSSION TITLE
Embed My results in an Html page.

### DISCUSSION THREAD

**Discussion by Abdulrhman5** [monkeytypegame/monkeytype#4435] [Ideas] [answered]
It would be great if there is an option to embed my result just like the ones here https://monkeytype.com/profile/SOMEONE on my website.

Thanks in advance

**Comment by ridemountainpig** [comment, marked as answer]
Hi @Abdulrhman5 , I have a project is MonkeyType Readme, it is a tool that transforms your MonkeyType typing data into SVGs, which you can put in your GitHub Readme or other place with image to show off your typing speed and accuracy.

MonkeyType Readme: https://monkeytype-readme.zeabur.app/
MonkeyType Readme Github: https://github.com/ridemountainpig/monkeytype-readme

**Reply by Abdulrhman5** [reply]
Hi @ridemountainpig. thank you for your response.
I don't know much about Frontend, so I have a question if I may :). 
If my website is a static HTML + CSS website. how often will the generated SVG be updated?

**Reply by ridemountainpig** [reply]
SVG will be update every time when your website load, the SVG url will get latest data every time.
https://monkeytype-readme.zeabur.app/Abdulrhman/nord_light

**Reply by Abdulrhman5** [reply]
Wow. super nice, thank you so much :).
